pages/proposals/language/spanish.json

```diff
@@ -6,9 +6,9 @@
     "background": {
       "header": "Bienvenido a la convocatoria de propuestas de MozFest 2017",
       "body": [
-        "Las sesiones de MozFest cubren un amplio abanico de temas, pero todas comparten dos cosas importantes: son interactivas e inclusivas. Preferimos talleres y exposiciones, antes que sesiones que solo se compongan de conferencias y presentaciones de diapositivas.",
+        "Las sesiones de MozFest cubren un amplio abanico de temas, pero todas comparten dos cosas importantes: son interactivas e inclusivas. Las sesiones con más éxito no son conferencias o presentaciones con diapositivas, sino que buscan desafiar y promover la participación de su audiencia.",
         "Las sesiones pueden albergar de 3 a 25 participantes, de diferentes edades y con distintas habilidades. Todos los participantes del festival pueden acudir a tu sesión.",
-        "Cualquiera puede enviar una propuesta para MozFest 2017. Cuando la hayas enviado, tu sesión entrará a formar parte de un proceso abierto en <a href=\"https://github.com/MozillaFoundation/mozfest-program-2017\" target=\"_blank\">GitHub</a>",
+        "Cualquiera puede enviar una propuesta para MozFest 2017. Cuando la hayas enviado, tu sesión entrará a formar parte de un proceso abierto en <a href=\"https://github.com/MozillaFoundation/mozfest-program-2017\" target=\"_blank\">GitHub</a>. Toda la información que envíes por aquí, excepto tu identificador de Twitter y correo electrónico, se publicará en GitHub. Mozilla puede ponerse en contacto contigo a través del correo en relación a la informaciónn que enviaste.",
```

Comment:
Commenting here so that we don’t forget.
`a la informaciónn` → `a la información`